gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
+distributionUrl=https\://gradleproxy\:gradleproxy@software.r3.com/artifactory/gradle-proxy/gradle-6.9.2-bin.zip
```

Comment:
This URL also contains a user account

Reply:
Of course, since this file is generated using the properties from the `wrapper` task.